### PR TITLE
Fix filtering on values containing spaces in @listing endpoint.

### PR DESCRIPTION
**Pull request by tinagerber**
In order to filter correctly in solr, terms containing spaces must be surrounded by quotation marks.

Jira: https://4teamwork.atlassian.net/browse/CA-2176

## Checklist

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

_Only applicable should be left and checked._

- API change:
  - [ ] Documentation is updated
  - [x] API Changelog entry (see [guide](https://4teamwork.atlassian.net/wiki/spaces/4TEAM/pages/451248812/API+Changelog+Guidelines))

### CODE REVIEW

File: opengever/api/solr_query_service.py [hunk 1]
```diff
@@ -30,19 +29,14 @@
 import Missing
 
 
-TO_ESCAPE = copy(SPECIAL_CHARS)
-TO_ESCAPE.append(' ')
-
-
 def filter_escape(term):
```
Comment (deiferni) [inline opengever/api/solr_query_service.py:32]: Could you add unit-tests for this function? It looks like it does not use any plone/zope dependencies and you can just use it in a unit test. Advantages of doing so:
- Document and test extensively how it behaves for edge cases (leading/trailing whitespace …)
- Have fast feedback during development (next time 😉 )

**Review by deiferni — APPROVED** [review]
LGTM 👍  but please add tests for edge cases so we gain more certainty this works correctly and as expected.
